TestCases/parallel_regression.py

```diff
@@ -185,6 +185,17 @@ def main():
     bluntbody.tol       = 0.00001
     test_list.append(bluntbody)
 
+    # Equivalent area NACA64-206
+    ea_naca64206           = TestCase('ea_naca64206')
+    ea_naca64206.cfg_dir   = "optimization_euler/equivalentarea_naca64206"
+    ea_naca64206.cfg_file  = "NACA64206_direct.cfg"
+    ea_naca64206.test_iter = 20
```

Comment:
this is what I meant above, use ITER= 4000 in the cfg and `ea_naca64206.test_iter = 40` here. 
In case you want to comment out options because they serve a purpose for easy switching between things please add an explaining comment why that is done and what expected changes are

Reply:
See [this comment](https://github.com/su2code/SU2/pull/1329#discussion_r684608624).